packages/zipkin/src/batch-recorder.js

```diff
@@ -105,8 +111,8 @@ class BatchRecorder {
 
   _writeSpan(id, span, isNew = false) {
     // TODO(adriancole) refactor so this responsibility isn't in writeSpan
-    if (!isNew && this.partialSpans.get(id) === 'undefined') {
-      // Span not found.  Could have been expired.
+    if (!isNew && typeof (this.partialSpans.get(id)) === 'undefined') {
```

Comment:
not sure why the extra parens are here. will still need to test this method to ensure it isn't lying as it was before